charts/sf-datapath-and-archival-gateway/values.yaml

```diff
@@ -56,6 +56,13 @@ nginx:
       - mountPath: /etc/logrotate.conf
         name: logrotate-config
         subPath: logrotate.conf
+    - name: dnsmasq
+      image: janeczku/go-dnsmasq:release-1.0.7
+      args:
+        - --listen
+        - "127.0.0.1:53"
+        - --default-resolver
+        - --append-search-domains
```

Comment:
this should be --enable-search instead of --append-search-domains
as it prints a log saying that it is deprecated